terraform/production/uht_selection_rules.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
I think this file has been removed because it's now in the common terraform repo

Reply:
delete from the branch?

Reply:
Yep, the DMS terraform for UH has been removed from here because it lives in [here](https://github.com/LBHackney-IT/aws-hackney-common-terraform/tree/master/applied_terraform/ProductionAPIs_aws_account/Housing_APIs) now

Reply:
cool

Reply:
removed